src/models/GOKU.jl

```diff
@@ -50,6 +73,10 @@ function apply_latent_out(decoder::Decoder{GOKU}, l̃)
     return ẑ₀, θ̂
 end
 
+@doc raw"""
+    diffeq_layer(decoder::Decoder{GOKU}, l̂, t)
+Uses decoder.diffeq's ODE solver to extrapolate the latent states (from the initial states and parameters l̂) to time t.
```

Comment:
I wouldn't say ODE solver because it is possible to use other differential equations different not just ODEs.
And also I wouldn't say that it extrapolates the latent states, I would say that that it integrates the differential equations contained in the `decoder`'s `diffeq` with the initial conditions (and possibly parameters) determined by `l̂`.

Reply:
So just DE Solver?

Reply:
But it's not integration....  It's using numerical methods to extrapolate the latent state based on the differential equations

Reply:
Some times it is called numerical integration. But maybe we should just call it solve the differential equations.